mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioSelector.java

```diff
@@ -90,6 +90,11 @@
     protected volatile Thread thread;
 
     /**
+     * Count down to 0 when the I/O thread starts and {@link #thread} is set to non-null.
+     */
+    final CountDownLatch startupLatch = new CountDownLatch(1);
```

Comment:
Yes, Netty was accessing this field directly from other classes.  This is a code diff from updating the new version